templates/etc/haproxy/global.cfg.j2

```diff
@@ -79,12 +79,12 @@
 {% for option in haproxy_global_option | default([]) %}
   {{ option }}
 {% endfor %}
+{% for line in haproxy_global_raw_options | default([]) %}
+  {{ line }}
+{% endfor %}
 {% for peers in haproxy_global_peers | default([]) %}
 peers {{ peers.name }}
 {% for peer in peers.peers | default([]) %}
   peer {{ peer.name }} {{ peer.listen }}
 {% endfor %}
 {% endfor %}
```

Comment:
Shouldn't a new line here be enough?

Reply:
No, because `peers` (line 83) is actually a different section of the config, and these lines need to go in the `global` section, so it needs to come before line 83.